Python/Product/Common/Strings.resx

```diff
@@ -1035,8 +1035,8 @@ Your web browser will not be opened.</value>
     <value>Unknown command '{0}', use '$help' for a list of commands.</value>
   </data>
   <data name="ReplCaption" xml:space="preserve">
-    <value>{0} Interactive</value>
-    <comment>{0} is the name of the Python environment</comment>
```

Comment:
you said you like space too, should we change it?

Reply:
yes. sorry missed that